sdk/loadtestservice/azure-resourcemanager-loadtestservice/src/test/java/com/azure/resourcemanager/loadtestservice/TestOrchestrator.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.loadtestservice;
+
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.util.Configuration;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.identity.DefaultAzureCredential;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import java.util.Random;
+import org.junit.jupiter.api.Test;
+
+public class TestOrchestrator extends TestBase {
+
+    private static final Random RANDOM = new Random();
+    private static final Region LOCATION = Region.US_WEST2;
+    private static final String RESOURCE_NAME = "loadtest-resource" + RANDOM.nextInt(1000);
+    private static final String QUOTA_BUCKET_NAME = "maxEngineInstancesPerTestRun";
+
+    private DefaultAzureCredential credential;
+    private AzureProfile profile;
+    private LoadTestManager loadTestManager;
+    private String resourceGroupName;
+
+    public void setupCredential() {
+        credential = new DefaultAzureCredentialBuilder().build();
+    }
+
+    public void setupProfile() {
+        profile = new AzureProfile(AzureEnvironment.AZURE);
+    }
+
+    public void prepareTests() {
+        setupCredential();
+        setupProfile();
+        resourceGroupName = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        loadTestManager = LoadTestManager
+            .configure()
```

Comment:
seems working fine.

can you add basic logging so we can see API calls in log?
e.g. https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/mediaservices/azure-resourcemanager-mediaservices/src/test/java/com/azure/resourcemanager/mediaservices/MediaServicesTests.java#L52